Test failure at EmbeddedSpringBootKeycloakServerXApplicationTests

Hi there!

This project looks very promising and I'm looking forward to it!

I just did a fresh git clone based on master and executing `mvn clean install` but it failed here:
```
07:14:33.763 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [com.github.thomasdarimont.keycloak.EmbeddedSpringBootKeycloakServerXApplicationTests]: class path resource [com/github/thomasdarimont/keycloak/EmbeddedSpringBootKeycloakServerXApplicationTestsContext.groovy] does not exist
07:14:33.765 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [com.github.thomasdarimont.keycloak.EmbeddedSpringBootKeycloakServerXApplicationTests]: no resource found for suffixes {-context.xml, Context.groovy}.
07:14:33.767 [main] INFO org.springframework.test.context.support.AnnotationConfigContextLoaderUtils - Could not detect default configuration classes for test class [com.github.thomasdarimont.keycloak.EmbeddedSpringBootKeycloakServerXApplicationTests]: EmbeddedSpringBootKeycloakServerXApplicationTests does not declare any static, non-private, non-final, nested classes annotated with @Configuration.
07:14:33.771 [main] DEBUG org.springframework.test.context.support.ActiveProfilesUtils - Could not find an 'annotation declaring class' for annotation type [org.springframework.test.context.ActiveProfiles] and class [com.github.thomasdarimont.keycloak.EmbeddedSpringBootKeycloakServerXApplicationTests]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.842 s <<< FAILURE! - in com.github.thomasdarimont.keycloak.EmbeddedSpringBootKeycloakServerXApplicationTests
[ERROR] com.github.thomasdarimont.keycloak.EmbeddedSpringBootKeycloakServerXApplicationTests  Time elapsed: 0.841 s  <<< ERROR!
java.lang.IllegalStateException: Unable to find a @SpringBootConfiguration, you need to use @ContextConfiguration or @SpringBootTest(classes=...) with your test
```

My Java version used on Mac:
```
openjdk version "11.0.7" 2020-04-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.7+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.7+10, mixed mode)
```

--- a/embedded-keycloak-server-plain/src/test/java/com/github/thomasdarimont/keycloak/EmbeddedSpringBootKeycloakServerXApplicationTests.java
+++ b/embedded-keycloak-server-plain/src/test/java/com/github/thomasdarimont/keycloak/EmbeddedSpringBootKeycloakServerXApplicationTests.java
@@ -3,7 +3,7 @@ package com.github.thomasdarimont.keycloak;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
-@SpringBootTest
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 class EmbeddedSpringBootKeycloakServerXApplicationTests {
 
     @Test

